packages/mini-cart/src/mini-cart.tsx

```diff
@@ -104,17 +104,23 @@ export function MiniCart( {
 	closeCart,
 	onRemoveProduct,
 	onRemoveCoupon,
+	checkoutLabel,
+	emptyCart,
 }: {
 	selectedSiteSlug: string;
 	cartKey: number | undefined;
 	goToCheckout: ( siteSlug: string ) => void;
 	closeCart: () => void;
 	onRemoveProduct?: ( uuid: string ) => void;
 	onRemoveCoupon?: () => void;
+	checkoutLabel?: string;
+	emptyCart: React.ReactNode;
```

Comment:
I think you wanted this to be optional, right? You have conditions below that imply it might not be required.

Reply:
I have updated it to be optional. Thanks

Reply:
Right, thanks @sirbrillig it should be better for the visibility. It still would work though as `React.ReactNode` is actually already optional. Basically is: `React.ReactNode =  boolean | React.ReactChild | React.ReactFragment | React.ReactPortal | null | undefined`

Reply:
> Basically is: React.ReactNode =  boolean | React.ReactChild | React.ReactFragment | React.ReactPortal | null | undefined

Yes, but there's a difference between `prop: string | undefined` and `prop?: string`. In the former, prop must still be specified, even if it's empty. In the latter, prop does not need to be set at all.

So `emptyCart: ReactNode` requires that a caller specify `emptyCart={ undefined }` if they do not want one, but `emptyCart?: ReactNode` lets it be omitted.